lib/routes/mirrormedia/filter.ts

```diff
@@ -0,0 +1,56 @@
+import { Route } from '@/types';
+
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+import { getArticle } from './utils';
+
+export const route: Route = {
+    path: '/category/:category',
+    categories: ['traditional-media'],
+    example: '/mirrormedia/category/political',
+    parameters: { category: '分类名' },
+    name: '分类',
+    maintainers: ['dzx-dzx'],
+    radar: [
+        {
+            source: ['mirrormedia.mg/category/:category'],
+        },
+    ],
+    handler,
+};
+
+async function handler(ctx) {
+    const { category } = ctx.req.param('category');
+    const rootUrl = 'https://www.mirrormedia.mg';
+
+    const response = await ofetch('https://adam-weekly-api-server-prod-ufaummkd5q-de.a.run.app/content/graphql', {
+        method: 'POST',
+        body: {
+            variables: {
+                take: ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 24,
+                skip: 0,
+                orderBy: { publishedDate: 'desc' },
+                filter: {
+                    state: { equals: 'published' },
+                    categories: { some: { slug: { equals: category } } },
+                },
+            },
+            query: 'fragment section on Section {\n  id\n  name\n  slug\n  state\n  __typename\n}\n\nfragment category on Category {\n  id\n  name\n  slug\n  state\n  __typename\n}\n\nfragment listingPost on Post {\n  id\n  slug\n  title\n  brief\n  publishedDate\n  state\n  sections(where: {state: {equals: "active"}}) {\n    ...section\n    __typename\n  }\n  categories(where: {state: {equals: "active"}}) {\n    ...category\n    __typename\n  }\n  isFeatured\n  __typename\n}\n\nquery ($take: Int, $skip: Int, $orderBy: [PostOrderByInput!]!, $filter: PostWhereInput!) {\n  postsCount(where: $filter)\n  posts(take: $take, skip: $skip, orderBy: $orderBy, where: $filter) {\n    ...listingPost\n    __typename\n  }\n}',
```

Comment:
https://docs.rsshub.app/joinus/advanced/script-standard#string
> Use [template literals](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Template_literals) for GraphQL queries as they make the code more concise and easy to read.

Reply:
But the query string is constant?

Reply:
Most likely yes